src/physics/lsm_driver.f90

```diff
@@ -202,6 +216,10 @@ subroutine lsm_var_request(options)
                          kVARS%humidity_2m, kVARS%temperature_2m])
         endif
 
+        if (options%physics%watersurface > 2) then
```

Comment:
can / should this be if `%watersurface == kWATER_LAKE` ?

Reply:
good point. Changed it